mobify/platform-scaffold
# [Integration Manager] Move `receiveFormInfo` into the Merlins Connector

Discovered that the `receiveFormInfo` is an implementation detail of the Merlin's connector. Moving it in and removing usage in the Demandware connector.

 **JIRA**: **N/A**
**Depends on**: https://github.com/mobify/platform-scaffold/pull/459 👈  Merge this first
 **Linked PRs**: https://github.com/mobify/platform-scaffold/pull/421

## Changes
- Move `receiveFormInfo` into the Merlins connector 
- Add placeholder `actions.js` for both connectors and hook the into the reducers to allow for connector-specific data in the store.

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Ensure you can go through home -> PLP -> PDP -> Add to cart -> Checkout
